swagger.json

```diff
@@ -1684,9 +1767,64 @@
           }
         }
       },
+      "put": {
```

Comment:
There should be a possibility to replace the process graph and maintain the existing ID.